Prox/Prox/PlaceCarouselViewController.swift

```diff
@@ -218,7 +226,7 @@ extension PlaceCarouselViewController: CLLocationManagerDelegate {
         mapView.region = MKCoordinateRegion(center: center, span: span)
 
         FirebasePlacesDatabase().getPlaces(forLocation: location).upon(DispatchQueue.main) { places in
-            self.placeCarousel.places = places.flatMap { $0.successResult() }
+            self.places = places.flatMap { $0.successResult() }
```

Comment:
we could set the text for the `numberOfPlacesLabel` here and then pass the places on to the `PlaceCarousel` for storage.

```
self.placeCarousel.places = places.flatMap { $0.successResult() }
headerView.numberOfPlacesLabel.text = "\(places.count) places"
```